torchvision/utils.py

```diff
@@ -191,34 +193,45 @@ def draw_bounding_boxes(
     elif image.size(0) not in {1, 3}:
         raise ValueError("Only grayscale and RGB images are supported")
 
+    if labels:
```

Comment:
We need to check these. As agreed. If labels are specified, all the boxes should have labels.